students/gene/project/node_db.py

```diff
@@ -0,0 +1,131 @@
+
+import sqlite3
+from sqlite3 import Error
+
+
+class Node:
+
+    def __init__(self, node_id, site_id='', management_ip='', console_id='', vendor_id='', platform_id='', os_id=''):
+        self.node_id = node_id
+        self.site_id = site_id
+        self.management_ip = management_ip
+        self.console_id = console_id
+        self.vendor_id = vendor_id
+        self.platform_id = platform_id
+        self.os_id = os_id
+
+
+    @property
+    def description(self):
+        return ("""Node: {}
+Site: {}
+Management IP: {}
+Console: {}
+Vendor: {}
+Platform: {}
+SW Version: {}""".format(self.node_id, self.site_id, self.management_ip,

```

Comment:
this would be a great place to use "f strings":

f"""Site: {self.site_id}"""

etc -- that is, you can put simple expression right in the brackets